src/common/Annotations.cs

```diff
@@ -0,0 +1,86 @@
+﻿// Copyright (c) Microsoft.  All Rights Reserved.  Licensed under the Apache License, Version 2.0.  See License.txt in the project root for license information.
+
+namespace System.Diagnostics.CodeAnalysis
+{
+    /// <summary>Specifies that null is allowed as an input even if the corresponding type disallows it.</summary>
+    [AttributeUsage(AttributeTargets.Field | AttributeTargets.Parameter | AttributeTargets.Property, Inherited = false)]
+    internal sealed class AllowNullAttribute : Attribute
```

Comment:
Can use [Nullable](https://www.nuget.org/packages/Nullable/) package

Reply:
Could do, but I don't think we should force a new package dependency for a single file.

Reply:
You know it's a development dependency right? If you take a dependency on Nullable, this will not cause anyone using your component to also depend on Nullable. So if that's the cause for concern, it's a non-issue. (If on the other hand, you want to minimize even  your build-time dependencies, then ignore me.)